posthog/api/user.py

```diff
@@ -45,6 +46,9 @@ def user(request):
             team.slack_incoming_webhook = data["team"].get("slack_incoming_webhook", team.slack_incoming_webhook)
             team.anonymize_ips = data["team"].get("anonymize_ips", team.anonymize_ips)
             team.session_recording_opt_in = data["team"].get("session_recording_opt_in", team.session_recording_opt_in)
+            if data["team"].get("plugins_opt_in", "nope") != "nope":
```

Comment:
Maybe it's better to use `None` as this nil value? You would only expect a boolean value here. Regardless #2111 would refactor this, so it might be fine to keep it.

Reply:
Changed. I tried to avoid this warning with the "nope", but it's too random to make sense: 
![image](https://user-images.githubusercontent.com/53387/97868714-75ebc500-1d10-11eb-9e49-1f6c10352683.png)

Reply:
I think you would need to do
```python
if data["team"].get("plugind_opt_in") is not None:
```

Reply:
Thanks! Fixed.